### PR TITLE
Use extrapolation for unreasonable/untabulated values for CO2 sequestration cases.

### PR DESCRIPTION
**Pull request by blattms**
Better alternative to #470 which recommendation from @totto82 incorporated. With this the case now seems to run. No downstream change needed.

### CODE REVIEW

**Comment by blattms** [comment]
Not sure whether we really want to limit the number of warnings as in bc6a3da. Maybe more is better?

**Comment by blattms** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks will test it. See also #472 which should take care of a similar issue for parametric functions.

**Comment by totto82** [comment]
> Not sure whether we really want to limit the number of warnings as in bc6a3da. Maybe more is better?

I suggest to keep it the way it is first without limiting the number of warnings. If its getting annoying we can add something like you suggests.

**Comment by blattms** [comment]
Just saw from your PR that opm-common is optional. Need to cater for that.

File: opm/material/common/UniformTabulated2DFunction.hpp [hunk 1]
```diff
@@ -185,22 +186,31 @@ class UniformTabulated2DFunction
...
         if (!applies(x,y))
         {
-            throw NumericalIssue("Attempt to get tabulated value for ("
-                                   +std::to_string(double(scalarValue(x)))+", "+std::to_string(double(scalarValue(y)))
-                                   +") on a table of extend "
-                                   +std::to_string(xMin())+" to "+std::to_string(xMax())+" times "
-                                   +std::to_string(yMin())+" to "+std::to_string(yMax()));
+            std::string msg = "Attempt to get tabulated value for ("
+                +std::to_string(double(scalarValue(x)))+", "+std::to_string(double(scalarValue(y)))
+                +") on a table of extend "
```
Comment (bska) [inline opm/material/common/UniformTabulated2DFunction.hpp:201]: Possible follow-up here: Is this supposed to say "extent" instead?

Comment (blattms) [reply opm/material/common/UniformTabulated2DFunction.hpp]: not from me but fixes anyway

**Comment by blattms** [comment]
jenkins build this please

**Comment by totto82** [comment]
Thanks. Looks good and works.

**Pull request merged**